.drone.yml

```diff
@@ -73,6 +94,9 @@ build:
       - cp $BIN/vic-ui* $BIN_TEMP_DIR
       - cp $BIN/appliance.iso $BIN_TEMP_DIR
       - cp $BIN/bootstrap.iso $BIN_TEMP_DIR
+      - mkdir $BIN_TEMP_DIR/ui
+      - cp -rf ui/installer/* $BIN_TEMP_DIR/ui
+      - sleep 1
```

Comment:
When I tested the the UI portion on my Vagrant box there was an issue w/ tar after the 'sleep 1' command where not some files were still being copied and thus tar failed. I assume that was the VM being slow so do you think it's ok to take that out?